rpc/const.go

```diff
@@ -13,4 +13,8 @@ var (
 	// proxies to indicate the address through which to route all network traffic
 	// via SOCKS5.
 	SocksProxyEnvVar = "SOCKS_PROXY"
+
+	// OnlySocksProxyEnvVar is the name of an environment variable used if all network
+	// traffic should be done through SOCKS5.
+	OnlySocksProxyEnvVar = "ONLY_SOCKS_PROXY"
```

Comment:
[q] Will there be an equivalent `ONLY_INTERNET`? Or, is the idea that having `SOCKS_PROXY` unset means "internet only"?

Reply:
I think that should be ok, + internet will almost always be prioritized/chosen even without this knob

Reply:
Discussed offline; this is also helpful for development, as we can force all traffic through the SOCKS proxy even with a WiFi connection (ssh still possible.)